manager/state/raft/raft.go

```diff
@@ -1092,6 +1151,11 @@ func (n *Node) reportNewAddress(ctx context.Context, id uint64) error {
 	if err != nil {
 		return err
 	}
+	if oldAddr == "" {
```

Comment:
Hmm, should we avoid calling `grpc.Dial` if `newPeer` is passed an empty address?